src/RadhydroShock/test_radhydro_shock.cpp

```diff
@@ -68,7 +68,7 @@ template <> struct RadSystem_Traits<ShockProblem> {
 template <> struct HydroSystem_Traits<ShockProblem> {
   static constexpr double gamma = gamma_gas;
   static constexpr bool reconstruct_eint = true;
-  static constexpr int nscalars = 1;       // number of passive scalars
+  static constexpr int nscalars = 0;       // number of passive scalars
```

Comment:
I set this to 1 just to make sure that having a passive scalar doesn't mess up the hydro solver. So this is important to keep as a test.

Reply:
But we should initialize the scalar to some constant value for this problem. It could be zero, or one, or pi.